dev_readme.md

```diff
@@ -1,30 +1,78 @@
 # How to start website
 
-#### 1. Download and install [Docker](https://www.docker.com/products/docker-desktop) for your operating system.
+// TODO: Merge into main readme. 
 
-#### 2. Clone the repository into your local directory.
```

Comment:
This is an excellent set of instructions - very asy to follow despite being a complex process.